numpy/core/__init__.py

```diff
@@ -75,8 +75,10 @@
 from . import fromnumeric
 from .fromnumeric import *
 from . import defchararray as char
+from . import records
 from . import records as rec
 from .records import record, recarray, format_parser
+from . import memmap
```

Comment:
`np.core.memmap` exists already: `np.core.memmap is np.memmap`. So this is not correct and could break some existing code possibly (ideally no one should use `np.core.<xxx>`, but they may be doing that today).